### ISSUE TITLE
Replace the alert dialog during deletion

### ISSUE THREAD

**Issue by almet** [Kinto/formbuilder#36]
Currently, when a widget is deleted, an `alert` dialog is shown to the users. We should find another mechanism instead.

I like the idea to change the label of the "delete" link to "are you sure"? and let the user click again, or display the user with a "cancel" link.

**Comment by QuentinRoy** [comment]
Another solution could be to delete the widget straight away, but then display a notification header that can be used to undo the deletion.E.g.:
<img width="463" alt="screen shot 2016-04-18 at 9 51 24 am" src="https://cloud.githubusercontent.com/assets/728801/14591887/26dfe19e-054b-11e6-859a-30a62b0adaab.png">
The issue with confirmation pop-ups or similar is that while they can succeed in avoiding action slips (e.g. "oops I clicked on the wrong button"), they fail in avoiding analysis errors (e.g. "oh no it was the other one that I needed to delete").

**Comment by almet** [comment]
> they fail in avoiding analysis errors (e.g. "oh no it was the other one that I needed to delete").

We could  add a text in the notification area where we say "Item 'blah' has been deleted".

**Comment by QuentinRoy** [comment]
Yep. I don't think it matters much where it is provided that it is not in the way (i.e. noticeable but ignorable) and that you can use it to undo what you just did.

**Comment by QuentinRoy** [comment]
Oh, and in this case, you may not want to make the notification disappear after sometime (cf #42).
